.env.local.example

```
@@ -0,0 +1,96 @@
+# OpenID Okta Secrets - DISABLED LOCALLY
+
+# The client ID for OpenID authentication from Okta
+# OPENID_CLIENT_ID=""
+# The client secret for OpenID authentication from Okta
+# OPENID_CLIENT_SECRET=""
+
+
+# S3 Configurations and Secrets
+
+# The S3 endpoint URL
+S3_ENDPOINT="http://minio:9000"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)